revocation/internal/crl/cache/crl.go

```diff
@@ -0,0 +1,183 @@
+package cache
+
+import (
+	"archive/tar"
+	"crypto/x509"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"time"
+)
+
+const (
+	// BaseCRL is the file name of the base CRL
+	BaseCRLFile = "base.crl"
+
+	// Metadata is the file name of the metadata
+	MetadataFile = "metadata.json"
+)
```

Comment:
Consider names like `PathBaseCRL` and `PathMetadata`

Reply:
Updated.